src/pymovements/utils/archives.py

```diff
@@ -41,6 +43,7 @@ def extract_archive(
         recursive: bool = True,
```

Comment:
can you add a `*,` before `recursive`? This is PR a breaking change either way, so we can make it more future proof.